src/en/discover/technology/index.html

```diff
@@ -114,21 +113,20 @@ <h2 class="h2 mb-12 lg:mb-16" id="block">Block storage</h2>
   <div class="grid lg:grid--cols-2 grid--gap-14 lg:grid--gap-28">
     <div>
       <p class="standout">
-        Access to block device images through Ceph’s thinly provisioned RADOS Block Device (RBD), with capabilities including snapshotting,
-        replication and striping for maximised availability.
+        Ceph Block Devices stripe block-device images over multiple objects in the Ceph Storage Cluster, where all objects are mapped to placement groups and distributed. The placement groups are then spread across separate ceph-osd daemons throughout the cluster. Thin-provisioned and snapshottable, Ceph Block Devices are an attractive option for cloud computing and virtualization.
```

Comment:
Ceph RBD (RADOS Block Device) block storage stripes virtual disks over objects in the Ceph storage cluster, distributing data and workload across all available devices for extreme scalability and performance. RBD disk images are thinly provisioned, support both read-only snapshots and writable clones, and can be asynchronously mirrored to remote Ceph clusters in other data centers for disaster recovery or backup, making Ceph RBD the leading choice for block storage in public/private cloud and virtualization environments.